### DISCUSSION TITLE
How do I cleanly uninstall Open-Shell please?

### DISCUSSION THREAD

**Discussion by annieb22** [Open-Shell/Open-Shell-Menu#1754] [General]
Please could anyone help me uninstall Open-Shell cleanly. I installed both the start menu as well as the file explorer but I had a few problems after updating my Windows 11 machine to the 23H2 build. The issues may or may not be related.

I uninstalled Open-Shell through Programs and Features and reinstalled just the Start Menu. However, I still have a Menu bar pop up in Windows File Explorer starting with 'Organise' and sometimes other items next to it. This menu bar isn't showing all of the time.

How do I compeletly uninstall Open-Shell? When I reinstalled the Start Menu only, I noticed that I didn't have to import my menu settings so something must have been left behind from the previous uninstall. I have subsequently rolled my PC back to version 22H2 (OS Build 22621.2428) but that menu bar is still there. I have the latest version of Open-Shell. 

This is how it looked when I had 23H2 on my PC.

![menubar](https://github.com/Open-Shell/Open-Shell-Menu/assets/33964872/57f87cc4-a644-42bf-9fb4-751d4b6d41aa)

**Comment by ge0rdi** [comment]
> I uninstalled Open-Shell through Programs and Features

This is the proper way to uninstall. If it didn't report any issues then `Open-Shell` is uninstalled.
You can check that `C:\Program Files\Open-Shell` folder doesn't exist anymore.

> I still have a Menu bar pop up in Windows File Explorer starting with 'Organise' and sometimes other items next to it.

I don't see how is this connected to `Open-Shell`. I don't think it is supposed to do anything with Explorer's menu.
Maybe you did change some Explorer settings?

>  I noticed that I didn't have to import my menu settings so something must have been left behind from the previous uninstall.

That is expected. `Open-Shell` keeps its settings in registry (even after uninstall). But this doesn't affect anything else than `Open-Shell`.

**Reply by annieb22** [reply]
Thank you ge0rdi for your reply. So the menu bar starting with 'Organise' doesn't come from Open-Shell?? Is that correct? 

I wonder where it comes from. Perhaps it was from the Windows 10 ribbon. The shop where I bought my PC from recently stated they installed a fresh installation of Windows 11 using a Windows 10 licence. 

I installed Winaero Tweaker too but I cannot see a setting about this. I'm not an expert but I don't think I did anything different to my old Windows 10 PC which had Winaero Tweaker and Classic Shell on it. I get that menu bar on that PC too.

**Reply by ge0rdi** [reply]
> So the menu bar starting with 'Organise' doesn't come from Open-Shell?? Is that correct?

Yes, I believe so.

That bar is in fact called `Command bar` and it looks like from Windows 7.
Maybe this will help:
https://www.askvg.com/enable-windows-7-command-bar-or-windows-10-ribbons-in-windows-11-file-explorer/

**Reply by annieb22** [reply]
Thank you for that link. I wonder how that Command bar has ended up on my PC. I have never installed Explorer Patcher and the PC was supposedly set up with a clean Windows 11 installation about a month ago. Most strange.

**Reply by Asad-13** [reply]
That command bar was in fact a part of Open Shell. It is called 'Classic Explorer' and you have the option to opt out of installing it while running setup.
I don't know how to uninstall it specifically though. Everytime I installed it, it was uninstalled along with the main Open Shell application.

**Reply by ge0rdi** [reply]
`Command bar` is for sure not part of `Classic Explorer`. I don't see anything related in the code.
Maybe installation of `Classic Explorer` somehow _triggers_ the bar in the system? Though I have never seen anything like that.

**Reply by bonzibudd** [reply]
There is an app called `OldNewExplorer` which restores the command bar. There is also a Winaero program called `Ribbon Disabler` which does the same thing.

**Comment by dutchmendesign** [comment]
I had to reset folder view settings using a .bat file.
https://www.tenforums.com/tutorials/17707-reset-folder-view-settings-default-windows-10-a.html
